Gems/Multiplayer/Code/Source/Components/NetBindComponent.cpp

```diff
@@ -26,11 +27,16 @@ namespace Multiplayer
 {
     void NetBindComponent::Reflect(AZ::ReflectContext* context)
     {
+        PrefabEntityId::Reflect(context);
+
         AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
         if (serializeContext)
         {
             serializeContext->Class<NetBindComponent, AZ::Component>()
-                ->Version(1);
+                ->Version(2)
+                ->Field("PrefabEntityId", &NetBindComponent::m_prefabEntityId)
+                ->Field("PrefabAssetId", &NetBindComponent::m_prefabAssetId)
```

Comment:
Nitpick: In the previous reflection block the fields started with a lower case while here they start with an upper case. For consistency stick with one or the other.
Just out of curiosity, why is this needed now that the Spawnable Entity Aliases are available?

Reply:
Converted to lower case. 
This info is needed for the multiplayer setup, each NetBindComponent needs to know which network spawnable it originated from and the index in the spawnable for the entity. 
That's how client side knows which entity to spawn when the server tells to do it